dev/monitoring/datarepo-monitoring.yaml

```diff
@@ -0,0 +1,108 @@
+          imagePullPolicy: Always
+          args:
+            - --stackdriver.project-id=broad-jade-dev      # project-id
+            - --prometheus.wal-directory=prometheus/wal
+            - --stackdriver.kubernetes.location=us-central1     # region
+            - --stackdriver.kubernetes.cluster-name=dev-master    # clustername
+            - --include={sd_export!="false",__name__!~"container_.*|scrape_.*|node_.*|prober_.*|instance:.*|code_.*|mixin_.*|cluster_.*",__name__=~".+"}
+            - --include={__name__=~"kube_node_status_condition"} # Used for cluster health alert
+          ports:
+            - name: stackdriver-exp
+              containerPort: 9091
+          volumeMounts:
+            - name: prometheus-datarepo-monitoring-kube-p-prometheus-db       #  in deployment yaml changes with helm deploy
+              mountPath: /prometheus
+      nodeSelector:
+        cloud.google.com/gke-nodepool: dev-node
+  grafana:
+    nodeSelector:
+      cloud.google.com/gke-nodepool: dev-node
+    rbac:
+      create: true
+      pspEnabled: true
+    plugins: ["doitintl-bigquery-datasource"]
+    env:
+      GF_USERS_AUTO_ASSIGN_ORG_ROLE: "Editor"
+      GF_AUTH_GOOGLE_ENABLED: "true"
+      GF_AUTH_GOOGLE_ALLOW_SIGN_UP: "true"
+      GF_AUTH_GOOGLE_CLIENT_ID: "970791974390-1581mjhtp2b3jmg4avhor1vabs13b7ur.apps.googleusercontent.com"
+      GF_AUTH_GOOGLE_SCOPES: "https://www.googleapis.com/auth/userinfo.profile https://www.googleapis.com/auth/userinfo.email"
+      GF_AUTH_GOOGLE_AUTH_URL: "https://accounts.google.com/o/oauth2/auth"
+      GF_AUTH_GOOGLE_AUTH_TOKEN: "https://accounts.google.com/o/oauth2/token"
+      GF_AUTH_GOOGLE_API_URL: "https://www.googleapis.com/oauth2/v1/userinfo"
+      GF_AUTH_GOOGLE_ALLOWED_DOMAINS: "broadinstitute.org verily.com"
+      GF_SERVER_DOMAIN: "grafana.datarepo-dev.broadinstitute.org"
+      GF_SERVER_ROOT_URL: "https://grafana.datarepo-dev.broadinstitute.org"
+    envFromSecret: "grafana-google-client-secret"
+    service:
+      annotations:
+        beta.cloud.google.com/backend-config: '{"ports": {"80":"datarepo-monitoring-backend-config"}}'
+      type: NodePort
+    ingress:
+      enabled: true
+      path: /*
+      hosts:
+        - grafana.datarepo-dev.broadinstitute.org
+        - grafana-global-dev.datarepo-dev.broadinstitute.org
+      annotations:
+        kubernetes.io/ingress.global-static-ip-name: grafana-k8-100
+        kubernetes.io/ingress.allow-http: "false"
+        networking.gke.io/managed-certificates: "grafana-gcp-managed-cert"
+      tls:
+        - hosts:
+            - grafana.datarepo-dev.broadinstitute.org
+            - grafana-global-dev.datarepo-dev.broadinstitute.org
+
+  alertmanager:
+    alertmanagerSpec:
+      nodeSelector:
+        cloud.google.com/gke-nodepool: dev-node
+    configFromSecret: alertmanager-jade-monitoring-prometheus-alertmanager
+    configFileName: alertmanager.yml
```

Comment:
Are we doing any custom alerting?

Reply:
we did but not anymore, I was testing it.